chart/README.md

```diff
@@ -520,12 +521,12 @@ Some example commands are shown below.
 #### External Database, Dynamic EFS, Dynamic ALB, and Secrets
    ```
    helm install mycluster
-   --set iq_server.serviceAccountName="nxlc-default-d95tfx8l-service"
+   --set iq_server.serviceAccountName=nxlc-service-account-name \
    --set serviceAccount.create=true
-   --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"="arn:aws:iam::552194422382:role/nxlc-default-d95tfx8l-role"
-   --set secret.arn="arn:aws:secretsmanager:us-east-2:552194422382:secret:nxlc-cluster/nxlc-default-d95tfx8l/nxlc-W999SV"
-   --set secret.license.arn="arn:aws:secretsmanager:us-east-2:552194422382:secret:nxlc-cluster/nxlc-default-d95tfx8l/nxlc_license-xpSvX5"
-   --set secret.rds.arn="arn:aws:secretsmanager:us-east-2:552194422382:secret:nxlc-cluster/nxlc-default-d95tfx8l/rds-FlwTg2"
+   --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"="arn:aws:iam::<AWS_ACCOUNT_ID>:role/<ROLE_NAME>" \
+   --set secret.arn="arn:aws:secretsmanager:<REGION>:<AWS_ACCOUNT_ID>:secret:<SECRET_NAME>" \
+   --set secret.license.arn="arn:aws:secretsmanager:<REGION>:<AWS_ACCOUNT_ID>:secret:<SECRET_NAME>"
+   --set secret.rds.arn="arn:aws:secretsmanager:<REGION>:<AWS_ACCOUNT_ID>:secret:<RDS_SECRET_NAME>" \
```

Comment:
Similar to above I'd suggest lowercase and no line endings

Reply:
Commit: decb4843589915c5637baf7df6ada1cbd8bbbe1e